api/server.spec.js

```diff
@@ -0,0 +1,139 @@
+const request = require('supertest');
+const server = require('./server.js');
+const db = require('../data/dbConfig.js');
+
+describe('route handlers', () => {
+    beforeEach( () => {
```

Comment:
You could do that the way you have it and I think that should work. Or you could do something like.
```suggestion
    beforeEach(async () => {
      await db.migrate.rollback()
      await db.migrate.latest()
      await db.seed.run()
    }
```
I think that looks a bit cleaner. But your way does work too 👍